### ISSUE TITLE
moving Notes and Contents fields of objects from MARC from tombstone data section to tabs

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#313]
See lengthy example:
[Helmingham herbal and bestiary, circa 1500](https://collections.britishart.yale.edu/catalog/orbis:9452785)

**Comment by flapka** [comment]
We’ve already agreed upon moving summary notes such as this to the curatorial description tab. That’ll take time, as they have to be edited one-by-one.

We haven’t however discussed “contents” notes, so I’m glad that @yulgit1 has raised the question. The Helmingham example is atypically long. This example is more representative of average length: https://collections.britishart.yale.edu/catalog/orbis:1322400

RBM would be open to the idea of creating a new tab category for these. For tab order, it might appear after curatorial description and before exhibition history. 

@KraigBinkowski, would this be desirable for Ref?

If we were to do this, it would be nice to implement a formatting adjustment to mimic what I’ve seen in other library catalogs, such as this Villanova example: https://library.villanova.edu/Find/Record/1644387#toc

There, the “ — “ (space hyphen hyphen space) syntax that separates each component of a contents note (MARC 505) is replaced with a hard return and a bullet, creating a more legible list view of the contents. Could we do that in Blacklight? 

Infrequently, records may have more than one contents note. I suppose these could appear in the same tab, separated by a small amount of white space (a couple of hard returns?)?

**Comment by yulgit1** [comment]
@flapka Reformatting contents, does this look OK:

![Screen Shot 2021-07-29 at 10 51 49 AM](https://user-images.githubusercontent.com/2125400/127514791-9a392cf8-05b7-47af-a9f4-302899e0ca4d.png)

**Comment by flapka** [comment]
@yulgit1 Yeah, I think such reformatting has a clear benefit for legibility. But it also occupies much more (vertical) screen space, so I think we’d want to implement this reformatting only if we moved contents notes to a tabbed structure. @KraigBinkowski, what do you think?

**Comment by yulgit1** [comment]
OK, what to call the tab, just "Contents"? And, where to put it in the ordered tabs, minding that tabs that don't exist aren't displayed?

Curatorial Comments
Curatorial Descriptions
Gallery Labels
Published Catalog Entries
Excerpts from this Work
Artist's Statement
Provenance
Exhibition History
Bibliography

**Comment by flapka** [comment]
For naming, I'd probably prefer plain "Contents". Alternatively, "Contents of the work". I want to avoid "Table of Contents" because that label would be at times misleading.

As for order: my position isn't fully resolved, but for now I lean towards placing it first.

**Comment by yulgit1** [comment]
resolved:
https://collections.britishart.yale.edu/catalog/orbis:9452785

**Comment by flapka** [comment]
I like it. If others, including @KraigBinkowski , feel the same, we could consider this closed.

**Comment by yulgit1** [comment]
Also in the issue: the Notes field.  I can move the  Notes to tabs now, but @flapka indicated above the Notes should go through editing first?

**Comment by edgartdata** [comment]
@flapka what would be an example of Notes?
